### PR TITLE
Use 8080 as port when started by mvn tomcat plugin

### PR DESCRIPTION
**Pull request by egh**
Set a default value for `maven.tomcat.port`.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -50,6 +50,7 @@
     <maven.build.timestamp.format>yyyyMMdd</maven.build.timestamp.format>
     <build.timestamp>${maven.build.timestamp}</build.timestamp>
     <build.counter>0</build.counter>
+    <maven.tomcat.port>8080</maven.tomcat.port>
```
Comment (sjtower) [inline pom.xml]: We set the port as a Tomcat runtime variable, not in the config xml:

`-Dmaven.tomcat.port=8234`

**Comment by egh** [comment]
@sjtower Sorry, I should have provided more background.

In the document https://plos.github.io/ambraproject/Quickstart-Guide.html it is suggested to start the ambra stack using:
```
wombat: mvn tomcat6:run -Dmaven.tomcat.port=8080 -Dwombat.configDir=/etc/ambra
rhino: mvn tomcat6:run -Dmaven.tomcat.port=8082 -Drhino.configDir=/etc/ambra
crepo: mvn tomcat6:run -Dmaven.tomcat.port=8081
```

My idea was to simplify this by setting each port recommended above as a default in the `pom.xml` for the maven tomcat plugin. Ideally, the user could use `mvn tomcat6:run` only for each component. (I will be filing a PR to use a default path for the ambra configuration dir).

The user can still use `-Dmaven.tomcat.port=8234` to override the port number, so no functionality is lost here, we are just setting a sensible default. See below for how this still works with a command line configured port.

```bash
$ mvn tomcat6:run -Dwombat.configDir=/home/egh/ambra/config/
[INFO] Scanning for projects...
...
INFO: Starting Coyote HTTP/1.1 on http-8080
^C
$ mvn tomcat6:run -Dwombat.configDir=/home/egh/ambra/config/ -Dmaven.tomcat.port=8234
[INFO] Scanning for projects...
...
INFO: Starting Coyote HTTP/1.1 on http-8234
```

**Comment by egh** [comment]
@sjtower I'm curious if you have any thoughts on my previous comment. Thank you!

**Comment by egh** [comment]
Thank you!

**Pull request merged**